site/homepage/layouts/partials/head.html

```diff
@@ -3,40 +3,37 @@
   <meta name="robots" content="all,follow">
   <meta name="googlebot" content="index,follow,snippet,archive">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ .Title | safeHTML }} :: {{ .Site.Title | safeHTML }}</title>
+  {{ $title_plain := .Title | markdownify | plainify }}
+  <title>{{ $title_plain }} :: {{ .Site.Title | safeHTML }}</title>
   <meta name="author" content="{{ .Site.Author.name }}" />
```

Comment:
This should be:
`<meta name="author" content="{{ .Param "author" }}" />`

Reply:
Yes, I didn't change it because it had been like that before. I will change that in the next PR that I have prepared for implementing the adivce for SEO that we got some time back from marketing ...